File: 프로그래머스/lv1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.java
class Solution {
    public int[] solution(int n, int m) {
        int[] answer = new int[2];
        int min = Math.min(n,m);
        int max = Math.max(n,m);
        if(max%min == 0){
            answer[0] = min;
            answer[1] = max;
            return answer;
        }
        for(int i=1; i<min; i++){
            if(min%i==0 && max%i==0)
                answer[0] = i;
        }
        answer[1] = min * (max/answer[0]);
        return answer;
    }
}

File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.js
function solution(n, m) {
    let min = Math.min(n, m);
    let max = Math.max(n, m);
    
    if(max%min === 0) return [min, max]
    
    for(var i=min; i>0 ; i--) {
        if(max%i===0 && min%i===0) return [i, min * (max/i)]
    }
}